examples/js/ThreeLoader.js

```diff
@@ -41,7 +41,7 @@ ThreeLoader.getThreeJsLoader = function getThreeJsLoader(format) {
     const deferredPromise = defer();
     // eslint-disable-next-line no-undef
     THREE = itowns.THREE;
-    loadScriptAsync('https://cdn.rawgit.com/mrdoob/three.js/r' + itowns.THREE.REVISION + '/examples/js/loaders/' + format + 'Loader.js')
+    loadScriptAsync('https://cdn.rawgit.com/mrdoob/three.js/r147/examples/js/loaders/' + format + 'Loader.js')
```

Comment:
Why you remove the dynamic loading?

Reply:
Because since r147, the `examples/js` folder is no longer. loaders are now in the form of js modules which can't be loaded that way. I tried to fix this in the Collada example (since it is the only one to use `ThreeLoader`) but we would need to be able to use import statements inside the examples, which did not work for some reason. I will open an issue on this specific point but as it is only for Collada, I figured we could have this temporary fix just to allow updating three for the moment.

Reply:
could you use `examples/jsm` instead of `examples/jsm`

Reply:
The Collada loader in jsm uses import statements which cannot be used outside modules. ThreeLoader does not load loaders as modules. ThreeLoader needs to be removed in my opinion, since it is only used to load collada loader in a way that is no longer supported by three (loading js scripts directly). Instead, we could refactor all the examples to use import statement (like what is done [here](https://github.com/mrdoob/three.js/blob/r154/examples/webgl_loader_collada.html) for instance). It would be closer to what users do, since importing iTowns and every other dependency with a `<script>` tag is not verry npm and webpack friendly. This would however take a bit of time to work on and would further delay threejs update, hence the temporary solution I suggest here.

Reply:
this example works but it needs to add `three` when the site is deployed

```html
<script type="importmap">
    {
        "imports": {
            "three": "../node_modules/three/build/three.module.js",
            "three/addons/": "../node_modules/three/examples/jsm/"
        }
    }
</script>

<script type="module">

    import { ColladaLoader } from 'three/addons/loaders/ColladaLoader.js';

    // ---------- CREATE A GlobeView FOR SUPPORTING DATA VISUALIZATION : ----------

    // Define camera initial position
    const placement = {
        coord: new itowns.Coordinates('EPSG:4326', 4.21655, 44.84415),
        range: 500,
        heading: 180,
        tilt: 60,
    };

    // `viewerDiv` will contain iTowns' rendering area (`<canvas>`)
    const viewerDiv = document.getElementById('viewerDiv');

    // Create a GlobeView
    const view = new itowns.GlobeView(viewerDiv, placement);

    // Setup loading screen and debug menu
    setupLoadingScreen(viewerDiv, view);
    const debugMenu = new GuiTools('menuDiv', view);



    // ---------- DISPLAY CONTEXTUAL DATA : ----------

    // Display ortho-images
    itowns.Fetcher.json('./layers/JSONLayers/Ortho.json')
        .then(function _(config) {
            config.source = new itowns.WMTSSource(config.source);
            view.addLayer(
                new itowns.ColorLayer('Ortho', config),
            ).then(debugMenu.addLayerGUI.bind(debugMenu));
        });

    // Display elevation data
    function addElevationLayerFromConfig(config) {
        config.source = new itowns.WMTSSource(config.source);
        view.addLayer(
            new itowns.ElevationLayer(config.id, config),
        ).then(debugMenu.addLayerGUI.bind(debugMenu));
    }
    itowns.Fetcher.json('./layers/JSONLayers/IGN_MNT_HIGHRES.json')
        .then(addElevationLayerFromConfig);
    itowns.Fetcher.json('./layers/JSONLayers/WORLD_DTM.json')
        .then(addElevationLayerFromConfig);

    const loadingManager = new itowns.THREE.LoadingManager();
    const colladaLoader = new ColladaLoader(loadingManager);

    const url = 'https://raw.githubusercontent.com/iTowns/iTowns2-sample-data/master/models/collada/building.dae';

    const fn = (collada) => {
        const model = collada.scene;

        // building coordinate
        const coord = new itowns.Coordinates(
            'EPSG:4326', 4.2165, 44.844, 1417,
        );

        model.position.copy(coord.as(view.referenceCrs));
        // align up vector with geodesic normal
        model.lookAt(model.position.clone().add(coord.geodesicNormal));
        // user rotate building to align with ortho image
        model.rotateZ(-Math.PI * 0.2);
        model.scale.set(1.2, 1.2, 1.2);

        // update coordinate of the mesh
        model.updateMatrixWorld();

        view.scene.add(model);
        view.notifyChange();
    };

    colladaLoader.load(url, fn);
</script>
```

Reply:
Thanks for the example suggestion. I used it in #2151. As the static import was merged in #2127 and is thus in the master branch, I think we can merge this PR to benefit the packages updates and fix the static import in another PR. WDYT @jailln and @gchoqueux ?

Reply:
ok for me